fms2_io/netcdf_io.F90

```diff
@@ -562,6 +568,7 @@ function netcdf_file_open(fileobj, path, mode, nc_format, pelist, is_restart, do
 
   integer :: nc_format_param
   integer :: err
+  integer :: IsNetcdf4=-999 !< Query the file for IsNetcdf4 in the event that the open for collective reads fails
```

Comment:
Because you are initializing this with a value, I think you should use some null value that is a parameter.  I also think `IsNetcdf4`  should be renamed if you are going to use an integer, because `IS` is typically used for logicals.  The style guide also asks that you use snake case for variable names, not camel case.

Reply:
I was just using the file attribute that is actually in our files.  Will change the variable name.  Also, no need to initialize the variable since it is never accessed before the NetCDF call to fill it.

Reply:
Changed IsNetcdf4 to netcdf4 and eliminated the initialization.